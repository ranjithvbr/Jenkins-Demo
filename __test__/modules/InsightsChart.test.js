test('1 + 1 = 2', () => {
  expect(1 + 1).toBe(2);
});

test('1 + 2 = 3', () => {
  expect(1 + 2).toBe(3);
});
