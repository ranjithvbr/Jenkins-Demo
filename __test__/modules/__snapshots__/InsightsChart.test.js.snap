// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Insight Charts for strict mode where the user data has  webAndAppActivity On locationHistory On and youtubeHistory On,  renders correctly 1`] = `undefined`;

exports[`Insight Charts renders correctly 1`] = `undefined`;
